sled-agent/src/illumos/link.rs

```diff
@@ -2,7 +2,7 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
```

Comment:
All changes to this file are basically taken from https://github.com/oxidecomputer/omicron/pull/1820/